components/script/task_manager.rs

```diff
@@ -141,6 +141,8 @@ impl TaskManager {
     task_source_functions!(self, file_reading_task_source, FileReading);
     task_source_functions!(self, font_loading_task_source, FontLoading);
     task_source_functions!(self, gamepad_task_source, Gamepad);
+    // FIXME(arihant2math): uncomment when geolocation is implemented.
+    // task_source_functions!(self, geolocation_task_source, Geolocation);
```

Comment:
Let's either add this now or removed the commented version until it's needed.

Reply:
How do I supress the clippy lint, using `#[allow]`/`#[expect]` doesn't work wth macros.